security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeCommon.php

```diff
@@ -154,8 +155,18 @@ public function setCa(string $uuid)
         $acme_ca = (string)$obj->ca;
         $this->ca = $acme_ca;
 
+        // Extract custom ACME CA URL
+        $acme_custom_ca = (string)$obj->custom_ca;
+        $this->custom_ca = $acme_custom_ca;
+
         // Add CA to acme arguments
-        $this->acme_args[] = LeUtils::execSafe('--server %s', $acme_ca);
+        if ($acme_ca == "custom") {
+            // Custom CA
```

Comment:
Please add a check for the "custom_ca" field, it must not be empty in this case. When empty, it should log an error and abort here.